internal/cli/base.go

```diff
@@ -873,5 +885,10 @@ short notation -p and -a.
 The current Waypoint server does not support snapshots. Rerunning the command
 with '-snapshot=false' is required, and there will be no automatic data backups
 for the server.
+`)
+
+	warnProjectFlagMismatch = strings.TrimSpace(`
+Warning: Currently in project directory for %q, but will operate 
+against specified project %q
```

Comment:
```suggestion
Warning: Currently in project directory for %q, but was requested 
to operate against specified project %q with '-project'
```

I think it'll be useful to be a bit more specific as to why the warning occurs.

Reply:
I'm going to add this in, as I told Izaak I'd work on merging his PRs while he is out this week.